server/embedded/build.gradle

```diff
@@ -0,0 +1,36 @@
+import org.labkey.gradle.util.BuildUtils
+
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'org.springframework.boot' version '2.3.3.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE' apply false
+}
+
+apply plugin: 'java'
+apply plugin: 'io.spring.dependency-management'
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ['src']
+        }
+    }
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-web:2.4.0'
+    implementation 'org.apache.tomcat.embed:tomcat-embed-core:9.0.34'
```

Comment:
This is what's defining which Tomcat is being used for embedding. It should be OK to split this from the top-level 8.5 version number.

But it seems useful to have a property for the Tomcat version we're using. Right now it's a mix of 9.0.34 and 9.0.37 which doesn't seem correct

Reply:
Why not embed the most recent Tomcat 9.0.x release? That would be 9.0.41.